### ISSUE TITLE
[user]. erratic behavior

### ISSUE THREAD

**Issue by flexoron** [infradig/trealla#360]
Trealla Prolog (c) Infradig 2020-2021, v1.18.31
?- ``[user].``
c.
cc.%
Error: syntax error, near '.%', operator expected 'cc.%
'
   true.
?- ``c.``
   true.
?- ``[user].``
c.
   true.
?- ``c.``
``true ;``
``true.``

Trealla Prolog (c) Infradig 2020-2021, v1.18.31
?- ``[user].``
c.
   true.
?- ``[user].``
c.
   true.
?- ``c.``
``true.``
?-

**Comment by flexoron** [comment]
Seems that every syntax error produces this result:

?- ``[user].``
c.
``c.,``
Error: syntax error, near '.', operator expected 'c.,
'
   true.
?- c.
   true.
?- ``[user].``
c.
   true.
?- ``c.``
``true ;``
``true.``
?-

**Comment by flexoron** [comment]
```
v1.19.17

$ tpl
?- [user].
c.
   true.
?- [user].
c.
   true.
?- c.
   true
;  true.
?-
```
new trealla version: error and error-free variant result the same.
related to #381 ?

**Comment by infradig** [comment]
CxProlog respects the multifile directive in [user] as well.

**Comment by flexoron** [comment]
Fine with me.
It means that v1.18.31 was wrong (and related to #381)

Copied from above:
```
Trealla Prolog (c) Infradig 2020-2021, v1.18.31
?- [user].
c.
true.
?- [user].
c.
true.
?- c.
true.
?-
```
and fixed in v1.19.17
```
?- c.
   true
;  true.
```
Left in the basket:
Case syntax error: unload_user  [user].
